data/graphql/schema.graphql

```diff
@@ -3832,7 +3832,7 @@ type Mutation {
   ): AgentStopPayload
 
   """
-  Create a new agent registration token.
+  Create a new unclustered agent token.
```

Comment:
For all the changes I've made to this file, I'm aware that these will need to be conducted in the main product source repo, which I'll start next week.